background-concealment-demo-py/BackgroundConcealmentDemo.ipynb

```diff
@@ -0,0 +1,268 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">- Line #2.    # so need to adopt model output for OpenCV image format. This is a an example how to user can implemenent and integrate
+ Line #2.    # so need to adopt model output for OpenCV image format. This is an example how to user can implemenent and integrate
</pre><p>Just a comment to highlight misprint</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TolyaTalamanov/g-api-collection/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>